framework/source/class/qx/ui/table/model/Filtered.js

```diff
@@ -68,6 +68,10 @@ search.addListener("changeValue", function(e) {
 
 this.getRoot().add(search, {top : 500, left : 10});
 </pre>
+ *
+ * @deprecated {6.0} You should use
+ * <a href='http://developer.mozilla.org/docs/Web/JavaScript/Reference/Objets_globaux/Array/filter' target='_blank'>Array.filter</a>
```

Comment:
I think you meant https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter - it looks like MDN localises the URLs and unless the locale is specified explicitly it's not reliable